site/static/examples/kind-with-registry.sh

```diff
@@ -22,10 +22,10 @@ containerdConfigPatches:
 EOF
 
 # connect the registry to the cluster network
-docker network connect "kind" "${reg_name}"
+docker network connect "kind" "${reg_name}" 2>/dev/null || true
```

Comment:
we should instead skip this step if the registry was already running. we can remember this from earlier in the script.
doing it this way means we ignore all errors.